CHANGES/7149.bugfix

```diff
@@ -0,0 +1 @@
+fix chunked upload timeouts with sock_read timeout set
```

Comment:
This needs to be formatted with RST. https://github.com/aio-libs/aiohttp/tree/master/CHANGES#alright-so-how-to-add-a-news-fragment.

Make sure it's consistent with other entries and describes the difference from the previous release.
Preview here: https://aiohttp--7150.org.readthedocs.build/en/7150/changes.html#bugfixes.